GUI/window.py

```diff
@@ -252,37 +252,49 @@ def __init__(self, **kwargs):
                                                     icon='run',
                                                     size_hint=(None, None),
                                                     pos_hint={
-                                                        'y': buttons_y_pos
+                                                        'y': self.buttons_y_pos
                                                     },
                                                     on_release=self.run_micro_instructions)
         self.debug_button = MDFillRoundFlatIconButton(text='Debug',
                                                       icon='android-debug-bridge',
                                                       size_hint=(None, None),
                                                       pos_hint={
-                                                          'y': buttons_y_pos
+                                                          'y': self.buttons_y_pos
                                                       },
                                                       on_release=self.run_micro_instructions_step)
         self.refresh_button = MDFillRoundFlatIconButton(text='Clear',
                                                         icon='refresh',
                                                         size_hint=(None, None),
                                                         pos_hint={
-                                                            'y': buttons_y_pos
+                                                            'y': self.buttons_y_pos
                                                         },
                                                         on_release=self.clear)
         self.save_button = MDFillRoundFlatIconButton(text='Save File',
                                                      icon='download',
                                                      size_hint=(None, None),
                                                      pos_hint={
-                                                         'y': buttons_y_pos
+                                                         'y': self.buttons_y_pos
                                                      },
                                                      on_release=self.open_save_dialog)
         self.pop_button = MDFillRoundFlatIconButton(text='Hex Keyboard',
                                                     icon='keyboard-outline',
                                                     size_hint=(None, None),
                                                     pos_hint={
-                                                        'y': buttons_y_pos
+                                                        'y': self.buttons_y_pos
                                                     },
                                                     on_release=self.run_window.open_keyboard)
+        self.loaded_file = MDIconButton(icon='file-check',
```

Comment:
disable buttons so that they are not clickable. This is to avoid confusing the user